tutorials/i18n/localization_using_csv.rst

```diff
@@ -0,0 +1,183 @@
+          
+::
+
+    # translation_example.gd
+    
+    extends Node2D
+
+    func _ready():
+        TranslationServer.set_locale("ja")      
+    
+        print(tr("GREET"))
+        
+        TranslationServer.set_locale("es")
+        
+        print(tr("BYE"))
+
+Plural translation
+------------------
+
+Plural translation can be tricky and is a common problem in text localization.
+
+In Godot, it is possible to provide plural translation with CSV files. To do this, 
+append the subscript ``[0]``, ``[1]``, ``[2]`` and so on for the keys dealing with plural translations.
+How far the index of the subscript goes depends on the required number of indices to satisfy the number of 
+plural forms of all locales. For example, Arabic has six plural forms. So if Arabic is one of 
+the supported locale in the CSV, the plural keys will have up to index ``[5]`` subscript.
+
++-------------+-----------------------+------------------------+------------------------------+
+| keys        | en                    | ja                     | ru                           |
++=============+=======================+========================+==============================+
+| KEY_HELLO   | Hello!                | こんにちは!            | Привет!                      |
++-------------+-----------------------+------------------------+------------------------------+
+| KEY_DAYS[0] | One day ago           | %d日前                 | %d день назад                |
++-------------+-----------------------+------------------------+------------------------------+
+| KEY_DAYS[1] | %d days ago           |                        | %d дня назад                 |
++-------------+-----------------------+------------------------+------------------------------+
+| KEY_DAYS[2] |                       |                        | %d дней назад                |
++-------------+-----------------------+------------------------+------------------------------+
+
+In ``GDScript``, use the :ref:`tr_n() <class_Object_method_tr_n>` function for plural translation. You don't need 
+to precise the subscript when using the key. Godot engine will concatenate the appropriate subscript for you 
+based on the current locale of ``TranslationServer`` and the argument ``n`` passed into the ``tr_n()`` function.
+For example (based on the above table):
+
+::
+
+    # translation_example.gd
+    
+    extends Node2D
+
+    func _ready():
+        TranslationServer.set_locale("en")      
+    
+        print(tr_n(1, "KEY_DAYS")) # Will print "One day ago".
+        print(tr_n(6, "KEY_DAYS")) # Will print "%d days ago".
+        print(tr_n(6, "KEY_DAYS") % 6) # Will print "6 days ago".
+		
+        TranslationServer.set_locale("ja")
+        
+        print(tr_n(1, "KEY_DAYS")) # Will print "%d日前".
+        print(tr_n(6, "KEY_DAYS")) # Will print "%d日前".
+        print(tr_n(6, "KEY_DAYS") % 6) # Will print "6日前".	
+
+Workflow
+--------
```

Comment:
I'm really not confident with this part regarding the workflow of CSV. I'd welcome any reviewer to offer their opinions whether I should remove it.